clients/src/test/java/org/apache/kafka/clients/admin/MockAdminClient.java

```diff
@@ -365,51 +366,6 @@ synchronized public DescribeTopicsResult describeTopics(Collection<String> topic
         return new DescribeTopicsResult(topicDescriptions);
     }
 
-    @Override
-    public DescribeConfigsResult describeConfigs(Collection<ConfigResource> resources) {
```

Comment:
Maybe just deprecate public method instead of removing it

Reply:
this method has default implementation (see https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/clients/admin/Admin.java#L340) which calls the variety ```describeConfigs(Collection<ConfigResource>, DescribeConfigsOptions)``` so it is ok to remove the new implementation.